integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionBadRecordLoggerTest.scala

```diff
@@ -53,8 +47,7 @@ class StandardPartitionBadRecordLoggerTest extends QueryTest with BeforeAndAfter
         " ',', 'QUOTECHAR'= '\"')")
     checkAnswer(
       sql("select count(*) from sales"),
-      Seq(Row(2)
```

Comment:
can move checkAnswer to single line in all places

Reply:
you mean in other class?